app/src/main/java/org/literacyapp/service/synchronization/AuthenticationJobService.java

```diff
@@ -31,19 +29,17 @@ public boolean onStartJob(JobParameters jobParameters) {
         Log.i(getClass().getName(), "onStartJob");
         if (!isScreenTurnedOff()){
             if (didTheMinimumTimePassSinceLastAuthentication()){
-                if (!rescheduleIfCpuIntensiveServicesAreRunning()){
-                    Intent authenticationIntent = new Intent(getApplicationContext(), AuthenticationActivity.class);
-                    authenticationIntent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-                    startActivity(authenticationIntent);
-                    Log.i(getClass().getName(), "The Authentication has been started.");
-                }
+                Intent authenticationIntent = new Intent(getApplicationContext(), AuthenticationActivity.class);
+                authenticationIntent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+                startActivity(authenticationIntent);
+                Log.i(getClass().getName(), "The Authentication has been started.");
             } else {
                 Log.i(getClass().getName(), "The Authentication was skipped because the minimum time since the last authentication did not pass yet.");
             }
         } else {
             Log.i(getClass().getName(), "The Authentication was skipped because the screen was turned off.");
         }
-
+        jobFinished(jobParameters, false);
```

Comment:
Here I didn't use a thread. Shall I use one here too? And how can I call jobFinished from another thread? Do I need to pass the AuthenticationJobService class to the thread as parameter?